addons/crm_livechat/models/mail_channel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from odoo import models, _
+
+
+class MailChannel(models.Model):
+    _inherit = 'mail.channel'
+
+    def _define_command_lead(self):
+        return {'help': _('Create a new lead')}
+
+    def _execute_command_lead(self, **kwargs):
+        partner = self.env.user.partner_id
+        msg = _('Something is missing or wrong in command')
```

Comment:
Instead of saying "Something is wrong", say instead "The correct way to use this command is bla bla bla"